test/unit/mock-filename-generator-spec.js

```diff
@@ -63,7 +63,26 @@ describe('mock filename generator', function() {
 
     var mockResponsePath = mockFilenameGenerator.getMockPath(prism, req);
 
-    assertPathEqual(mockResponsePath, 'mocks/foo/_is_this_url_really=that_237a2df1a4dfefe26d84b37103583200b8cd9c48.json');
+    assertPathEqual(mockResponsePath, 'mocks/foo/_is_this_url_237a2df1a4dfefe26d84b37103583200b8cd9c48.json');
```

Comment:
if you set `ignoreParameters = true` it should ignore all parameters, otherwise you need to specify them in in detail (by array/regex, see my other new test). So this test was expecting a wrong url.

The reason is that the implementation of `prismUtils.filterUrl()` was not correct, that's why the first parameter was never filtered... see my issue #67 for more details.